### DISCUSSION TITLE
Upgrade tools in Gradle builds

### DISCUSSION THREAD

**Discussion by jochenberger** [renovatebot/renovate#12422] [Request Help] [answered]
Hi,

renovate doesn't seem to be able to create PRs for the PMD tool version in Gradle builds.
Is there a way to extend the behavior or am I missing something?

This is how it looks in the build file btw. See also https://docs.gradle.org/current/dsl/org.gradle.api.plugins.quality.PmdExtension.html#org.gradle.api.plugins.quality.PmdExtension:toolVersion.
```groovy
pmd {
  toolVersion = '6.36.0'
}
```

**Comment by rarkins** [comment, marked as answer]
No, please create a feature request with full requirements/links to documentation, etc.

**Reply by jochenberger** [reply]
Done, thanks: https://github.com/renovatebot/renovate/issues/12424
